vendor/github.com/containerd/continuity/fs/copy_windows.go

```diff
@@ -17,19 +17,60 @@
 package fs
 
 import (
+	"errors"
+	"fmt"
 	"io"
 	"os"
 
-	"github.com/pkg/errors"
+	winio "github.com/Microsoft/go-winio"
+	"golang.org/x/sys/windows"
 )
 
-func copyFileInfo(fi os.FileInfo, name string) error {
+const (
+	seTakeOwnershipPrivilege = "SeTakeOwnershipPrivilege"
+)
+
+func copyFileInfo(fi os.FileInfo, src, name string) error {
```

Comment:
For others; also included in this diff; https://github.com/containerd/continuity/pull/188